Builds/containers/ubuntu-builder/Dockerfile

```diff
@@ -1,36 +1,28 @@
-ARG DIST_TAG=18.04
+ARG DIST_TAG=20.04
 FROM ubuntu:$DIST_TAG
 ARG GIT_COMMIT=unknown
 ARG CI_USE=false
 LABEL git-commit=$GIT_COMMIT
 
 # install/setup prerequisites:
 COPY ubuntu-builder/ubuntu_setup.sh /tmp/
-COPY shared/build_deps.sh /tmp/
 COPY shared/install_cmake.sh /tmp/
-COPY shared/install_boost.sh /tmp/
 RUN chmod +x /tmp/ubuntu_setup.sh && \
-    chmod +x /tmp/build_deps.sh && \
-    chmod +x /tmp/install_boost.sh && \
     chmod +x /tmp/install_cmake.sh
 RUN /tmp/ubuntu_setup.sh
 
-RUN /tmp/install_cmake.sh 3.16.1 /opt/local/cmake-3.16
+RUN /tmp/install_cmake.sh 3.16.3 /opt/local/cmake-3.16
 RUN ln -s /opt/local/cmake-3.16 /opt/local/cmake
 ENV PATH="/opt/local/cmake/bin:$PATH"
-# also install min supported cmake for testing
+# TODO: Install latest CMake for testing
 RUN if [ "${CI_USE}" = true ] ; then /tmp/install_cmake.sh 3.9.0 /opt/local/cmake-3.9; fi
 
-RUN /tmp/build_deps.sh
-ENV PLANTUML_JAR="/opt/plantuml/plantuml.jar"
-ENV BOOST_ROOT="/opt/local/boost/_INSTALLED_"
 ENV OPENSSL_ROOT="/opt/local/openssl"
 
 # prep files for package building
-RUN mkdir -m 777 -p /opt/rippled_bld/pkg/debian
-RUN update-alternatives --set gcc /usr/bin/gcc-8
+RUN update-alternatives --set gcc /usr/bin/gcc-11
+RUN mkdir -m 777 -p /opt/rippled_bld/pkg/
```

Comment:
I'd like to see almost every `RUN` statement in this Dockerfile, including these, moved to `ubuntu_setup.sh`. Best to have all the commands in one place, that another person can execute themselves without having to build a Docker image. My personal workflow when developing these install scripts is to log in interactively to a Docker container and copy bits of the script over via tmux; checking the output; and adding, removing, and changing bits as needed.